pkg/syncer/fullsync.go

```diff
@@ -720,26 +720,28 @@ func volumeInfoCRFullSync(ctx context.Context, metadataSyncer *metadataSyncInfor
 			} else if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload &&
 				IsPodVMOnStretchSupervisorFSSEnabled {
 				pv := volumeIdTok8sPVMap[volumeID]
-				pvc, err := metadataSyncer.pvcLister.PersistentVolumeClaims(
-					pv.Spec.ClaimRef.Namespace).Get(pv.Spec.ClaimRef.Name)
-				if err != nil {
-					log.Warnf("Failed to get pvc for namespace %s and name %s. err=%+v",
-						pv.Spec.ClaimRef.Namespace, pv.Spec.ClaimRef.Name, err)
-					continue
-				}
-				pvcCapacity := pvc.Status.Capacity[v1.ResourceStorage]
-				if pvc.Spec.StorageClassName != nil {
-					err = volumeInfoService.CreateVolumeInfoWithPolicyInfo(ctx, volumeID, pvc.Namespace,
-						scNameToPolicyIdMap[*pvc.Spec.StorageClassName], *pvc.Spec.StorageClassName, vc, &pvcCapacity)
+				if pv.Spec.ClaimRef != nil {
```

Comment:
Use inverted if statements for better readability https://www.geeksforgeeks.org/writing-clean-else-statements/